JSONArray.java

```diff
@@ -1159,10 +1157,8 @@ public String toString() {
      * @throws JSONException
      */
     public String toString(int indentFactor) throws JSONException {
-        StringWriter sw = new StringWriter();
-        synchronized (sw.getBuffer()) {
```

Comment:
This reverts a change related to https://github.com/stleary/JSON-java/issues/54

Basically, now that we use StringBuilder, which is not synchronizing on a buffer, instead of StringWriter, which does, the rationale for synchronizing no longer applies.